test/e2e/e2e.go

```diff
@@ -143,6 +144,13 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 		e2elog.Logf("kube-apiserver version: %s", serverVersion.GitVersion)
 	}
 
+	// Obtain the (primary if dual stack) ip family of the cluster
```

Comment:
I think we need another comment here (and in the TestConfig struct) about why we only check the ClusterIP of the `kubernetes` service to determine the IP family of the cluster. Maybe a TODO to also check other resources like nodes and pods?

Reply:
done